CMakeLists.txt

```diff
@@ -79,10 +83,6 @@ if(GCOV)
   set(linux_flags ${linux_flags} -g -O0 -fprofile-arcs -ftest-coverage -fno-strict-aliasing -pg)
 endif()
 
-set(windows_release_config /GL /O2 /Gy /Oi /Ob2 /Ot /Oy /GT /MD)
```

Comment:
Do we not need these compiler flags? (I am not sure what they do offhand)

Reply:
These flags are set in the CMakeSettings.json file. Setting this stuff here was causing problems between the various configs

Reply:
Does build-windows.bat need to reference this somewhere?

Reply:
It seems to be automagic CMake functionality

Reply:
Where is CMakeSettings.json? It isn't in the repo? These flags are required in some form